test/functional/tests/stress/test_stress_without_io.py

```diff
@@ -239,8 +248,9 @@ def test_stress_reload_module(cache_mode):
           - No data corruption.
     """
     with TestRun.step("Prepare cache and core. Create test file and count it's checksum."):
-        cache, core, md5_before_load, size_before_load, permissions_before_load = \
-            prepare_with_file_creation(cache_mode)
+        cache, core, file = prepare_with_file_creation(cache_mode)
+        file_before = file.get_properties()
```

Comment:
does not work - cannot get file properties if the core is unmounted (and it is unmounted in prepare_with_file_creation)

Reply:
Moved to 'get_properties' before unmounting core and returned.